Examples/CppCMake/Documentation.rst

```diff
@@ -17,86 +19,105 @@ Here is our SimpleITK C++ example file **sitk_example.cxx**.
+CMake attempts to find these files automatically, but it is unlikely to find
+them unless SimpleITK and ITK are installed into the system.
 
-If SimpleITK has been built using the SuperBuild process, as described in :doc:`building`, then
-these required directories can be found in the SuperBuild location.  The SimpleITK_DIR variable
-should refer to the directory SimpleITK-build within the SuperBuild location, and the ITK_DIR
-variable should refer to the directory ITK-build in the SuperBuild location.
+If SimpleITK has been built using the SuperBuild process, as described in
+:doc:`building`, then these required directories can be found in the SuperBuild
+location.  The SimpleITK_DIR variable should refer to the directory
+SimpleITK-build within the SuperBuild location, and the ITK_DIR variable should
+refer to the directory ITK-build in the SuperBuild location.
 
 Here are what the settings in a CMakeCache.txt file should look like::
 
     //The directory containing a CMake configuration file for ITK.
-    ITK_DIR:PATH=/home/dchen/SuperBuild/ITK-build
+    ITK_DIR:PATH=/home/dchen/SimpleITK-build/ITK-build
 
     //The directory containing a CMake configuration file for SimpleITK.
-    SimpleITK_DIR:PATH=/home/dchen/SuperBuild/SimpleITK-build
+    SimpleITK_DIR:PATH=/home/dchen/SimpleITK-build/SimpleITK-build
 
 Compiling using CMake
 #####################
-A SimpleITK program can also be compiled by running CMake.  Here is the command to perform the
-build in the current directory::
+A SimpleITK program can also be compiled by running CMake.  Here is the command
+to perform the build in the current directory::
 
     cmake --build . --config Release
 
-The 'config' parameter could also be set to Debug, RelWithDebInfo or MinSizeRel.  On
-Unix-type systems Release is the default.  On Windows the config must be specified explicitly.
+The 'config' parameter could also be set to Debug, RelWithDebInfo or
+MinSizeRel.  On Unix-type systems Release is the default.  On Windows the
+config must be specified explicitly.  We recommend Release or MinSizeRel,
+since SimpleITK compiled in Debug mode is several orders of magnitude slower.
 
-Of course traditional methods can be used to build a SimpleITK program (e.g. make on Unix, or
-Visual Studio on Windows), but it is often convenient to be able to use the same build command on
-all system types.
+Of course traditional methods can be used to build a SimpleITK program (e.g.
+make on Unix, or Visual Studio on Windows), but it is often convenient to be
+able to use the same build command on all system types.
 
 
-Using CMake on Windows
-######################
+Using CMake-gui on Windows
+##########################
 
-The Windows version of CMake includes a graphical-user-interface (GUI) version of the program,
-in addition to the standard command-line CMake.  Here is what the CMake GUI looks like:
+The Windows version of CMake includes a graphical-user-interface (GUI) version
```

Comment:
The GUI is not specific to windows (I use it on OSX). Possibly say "In addition to the standard command-line CMake, a graphical-user-interface (GUI) version of CMake is also available.

Reply:
done